intranet/templates/signage/pages/bus.html

```diff
@@ -98,53 +106,113 @@
     svg text.extra-small {
         font-size: 90%;
     }
+
+    .bus-col {
+        display: inline-block;
+        width: 33%;
+        height: 100%;
+        vertical-align: top;
+    }
+
+    .wrapper-1 {
+        width: 100%;
+        height: 50px;
+    }
+
+    .wrapper-2 {
+        margin-top: 15px;
+        width: 100%;
+        height: 100%;
+    }
+
+    .welcome-text {
+        font-size: 50px;
+        padding: 40px 20px;
+    }
+
+    .bus-header {
+        font-size: 30px;
+        padding: 15px;
+    }
+
+    .list-bus {
+        font-size: 20px;
+        padding: 15px;
+    }
     </style>
 {% endblock %}
 
 {% block body %}
-    {% verbatim %}
-    <script type="text/template" id="personal-status"></script>
-    <script type="text/template" id="route-view">
-        <span class="route-name"><%= route_name %></span>
-        <br>
-        <% if (bus_number) { %>
-            <span class="bus-number"> #<%= bus_number %> </span>
-        <% } else { %>
-            <span class="bus-number"> No bus number recorded. </span>
-        <% } %>
-        <% if (isAdmin) { %>
-            <select name="status" id="<%= id %>">
-                <% for (var key in label_status_strings) { %>
-                    <option value="<%= key %>" <% if (key === status) print("selected") %>> <%= label_status_strings[key].name %> </option>
-                <% } %>
-            </select>
-        <% } %>
-    </script>
-    <script type="text/template" id="status-group-view">
-        <% console.log(name, empty_text, collection)%>
-        <h2 class="status"> <%= name %> </h2>
-        <% if (collection.length === 0) { %>
-            <h3 class="empty"> <%= empty_text %> </h3>
-        <% } %>
-    </script>
-    <script type="text/template" id="action-button-view">
-    </script>
-    <script type="text/template" id="search-widget-view">
-        <div class="search-widget">
-            <i class="back-button fas fa-arrow-left"></i><select name="buses" id="buses"></select>
+    {% if time == "morning" %}
+        <h1 class="welcome-text">Good morning!</h1>
+        <div class="wrapper-1">
```

Comment:
I would make the class names a bit more descriptive than `wrapper-1` and `wrapper-2`. Maybe `bus-heading` and `bus-body`?